api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/AbstractReferenceHandlingTranslator.java

```diff
@@ -109,6 +111,13 @@ protected Reference createPractitionerReference(@NotNull Provider provider) {
 		return reference;
 	}
 	
+	protected Reference createOrderReference(@NotNull Order order) {
+		if (order instanceof DrugOrder) {
+			return null;
```

Comment:
We actually shouldn't return null if it's a `DrugOrder`. I'd just handle it as usual anyways.